GiveNGo/GiveNGo
# Conduct

## Description
* Updated Code of Conduct page with text component

## What I changed
* Added card and text components with code of conduct text
* added styling 

## How to test
* `npm start`
* navigate to Sign Up page, once you click ‘sign up’ you should redirect to review the GNG code of conduct
